docs/content/release_notes/changelog_1.8.0.md

```diff
@@ -1,6 +1,6 @@
 # 1.8.0 (December 2021)
 
-IBM® block storage CSI driver 1.8.0 adds new support and enhancements:
+IBM® block storage CSI driver 1.8.0 added new support and enhancements:
 - New support and enhancements for IBM Spectrum Virtualize family storage systems:
```

Comment:
```suggestion
- New support and enhancements for Spectrum Virtualize family storage systems:
```

Reply:
unless there is a good reason, we don't got back and change the content of the older changelog content. this does not warrant a change.

Reply:
ok (thought we do because of the 1.4.0 "added" change)